src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/.template.config/localize/templatestrings.en.json

```diff
@@ -36,7 +36,7 @@
   "symbols/CalledApiUrl/description": "URL of the API to call from the web app. This option only applies if --auth SingleOrg, --auth MultiOrg or --auth IndividualB2C without and ASP.NET Core host is specified.",
   "symbols/CallsMicrosoftGraph/description": "Specifies if the web app calls Microsoft Graph. This option only applies if --auth SingleOrg or --auth MultiOrg is specified.",
   "symbols/CalledApiScopes/description": "Scopes to request to call the API from the web app. This option only applies if --auth SingleOrg, --auth MultiOrg or --auth IndividualB2C without and ASP.NET Core host is specified.",
-  "symbols/UseProgramMain/displayName": "Do not use _top-level statements",
+  "symbols/UseProgramMain/displayName": "Do not _use _top-level statements",
```

Comment:
Somehow ended up with two underscores here, I don't think that's intentional?

Reply:
```suggestion
  "symbols/UseProgramMain/displayName": "Do not use _top-level statements",
```